### PR TITLE
pythonlib v0.15.1

### PR DESCRIPTION
**Pull request by tov**
OCaml 5 support

### CODE REVIEW

File: packages/pythonlib/pythonlib.v0.15.1/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+homepage: "https://github.com/janestreet/pythonlib"
+bug-reports: "https://github.com/janestreet/pythonlib/issues"
+dev-repo: "git+https://github.com/janestreet/pythonlib.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/pythonlib/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"      {>= "4.11.0"}
```
Comment (kit-ty-kate) [inline packages/pythonlib/pythonlib.v0.15.1/opam:13]:
```suggestion
  "ocaml"      {>= "4.14.0"}
```

Comment (kit-ty-kate) [reply packages/pythonlib/pythonlib.v0.15.1/opam:13]: Actually maybe i could’ve used `[%%if ocaml_version ...` in my PR. Could you add those where necessary if you think this is needed? (in the current state this release skips OCaml 4.13)

Comment (tov) [reply packages/pythonlib/pythonlib.v0.15.1/opam:13]: Nevermind, I figured it out.

**Comment by tov** [comment]
Okay, let's see how this does.

**Comment by tov** [comment]
It looks like the problem here is that `pyml < 20220322` doesn't work on OCaml 5.0. Would it make sense to add an upper bound of `< "5.0"` to the older `pyml` versions, rather than raising the lower bound for `pythonlib`?

**Comment by kit-ty-kate** [comment]
Done in #22315. Thanks.

**Pull request merged**